Meta/Lagom/Tools/CodeGenerators/StateMachineGenerator/CMakeLists.txt

```diff
@@ -2,5 +2,4 @@ set(SOURCES
     main.cpp
 )
 
-add_executable(StateMachineGenerator ${SOURCES})
-target_link_libraries(StateMachineGenerator LagomCore)
+lagom_tool(StateMachineGenerator ${SOURCES})
```

Comment:
I don't think this was intentional. `SOURCES` is taken from the calling scope(s) already.

Reply:
Indeed it wasn't. incomplete transformation on this file. fixed in d6c4ea86018ee722aae5b0b4d34e81e28313b4c6